client/src/components/pages/Appointments.js

```diff
@@ -1,17 +1,19 @@
 import React from "react";
-import { Table } from 'react-bootstrap';
+import { Table, ButtonGroup, Button } from 'react-bootstrap';
 
 
 function Appointment() {
   return (
-    <div>
+    <div className="container">
+
+    <br></br>
```

Comment:
For future reference, it's generally better to put spacing/margin/padding into css rather than markup like this. That way it's easier to tweak that spacing in the future by only updating the styles in one spot instead of searching for multiple `<br>`s